### PR TITLE
1.2.0

### PR DESCRIPTION
**Pull request by lakhoune**
Develop and master have not been merged for a long time. Consequently, the master is still using ant and Travis. This pull request introduces Gradle which has been introduced on develop for quite some time now

### CODE REVIEW

File: .github/workflows/docker-CI.yml [hunk 1]
```diff
@@ -2,10 +2,29 @@ name: Docker Build and Push
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop,'1.2.0' ]
```
Comment (Aran30) [inline .github/workflows/docker-CI.yml]: 1.2.0 not needed right?

File: etc/launcher-configuration.ini [hunk 2]
```diff
@@ -0,0 +1,15 @@
+port = 9011
```
Comment (Aran30) [inline etc/launcher-configuration.ini]: this file is redundant

File: mobsos-data-processing/src/main/java/i5/las2peer/services/mobsos/dataProcessing/MobSOSDataProcessingService.java [hunk 3]
```diff
@@ -54,6 +54,7 @@ public class MobSOSDataProcessingService extends Service {
 	private Set<String> triggerFunctions = new HashSet<String>();
 	private ArrayList<BotMessage> botMessages = new ArrayList<BotMessage>();
 	private ArrayList<String> xAPIstatements = new ArrayList<String>();
+	//private boolean sendStatementsToBots = true; // True if xAPI statements should also be sent to the social bot manager service
```
Comment (Aran30) [inline mobsos-data-processing/src/main/java/i5/las2peer/services/mobsos/dataProcessing/MobSOSDataProcessingService.java]: why is this commented out?

**Review by Aran30 — CHANGES_REQUESTED** [review]
See comments

File: .github/workflows/docker-CI.yml (hunk 1 above)
Comment (lakhoune) [reply .github/workflows/docker-CI.yml]: true

File: etc/launcher-configuration.ini (hunk 2 above)
Comment (lakhoune) [reply etc/launcher-configuration.ini]: True, interesting that it was not in the gitignore file

File: mobsos-data-processing/src/main/java/i5/las2peer/services/mobsos/dataProcessing/MobSOSDataProcessingService.java (hunk 3 above)
Comment (lakhoune) [reply mobsos-data-processing/src/main/java/i5/las2peer/services/mobsos/dataProcessing/MobSOSDataProcessingService.java]: I am not sure. This was introduced in https://github.com/rwth-acis/mobsos-data-processing/commit/1bcbd2d9f57afc8d25c9763ecbd3cce56bbba1eb

**Review by Aran30 — APPROVED** [review]

**Pull request merged**